joystickwake

```diff
@@ -345,6 +345,7 @@ DEFAULT_WAKERS = [
     ExecWaker('gnome-screensaver-command', '--deactivate', name="GNOME"),
     ExecWaker('mate-screensaver-command', '--poke', name="MATE"),
     ExecWaker('xfce4-screensaver-command', '--poke', name="Xfce"),
+    ExecWaker('qdbus', 'org.freedesktop.ScreenSaver', '/ScreenSaver', 'SimulateUserActivity', name='org.freedesktop.ScreenSaver'),
```

Comment:
Okay. I'm surprised that this command can't be used in GNOME/etc since it's supposed to be standard. I guess whether it can bypass a screensaver already running is not defined.

> > Just to confirm: is the expected behaviour that if the screensaver is running/the display is sleeping, pressing a button on a joypad would wake the screen?
> 
> Yes, and this command fails to do it on KWin (KDE Plasma), even when I have "Allow unlocking without password" set to a sufficiently long duration.
> 
> However, it turns out that the command does keep KWin's locker from activating in the first place, so I'm inclined to include it in joystickwake despite KWin's after-blank bug.
> 
> > I have been using this alias for some time (I would love to not have to)
> 
> Instead of that infinite loop, why not configure joystickwake to run the dbus command? Put this in $HOME/.config/joystickwake/joystickwake.conf :
> 
> ```
> command = qdbus org.freedesktop.ScreenSaver /ScreenSaver SimulateUserActivity
> ```

I am happy to do that, but I submitted this PR so others would have it without having to make a config file.

Reply:
> KDE bug reported: https://bugs.kde.org/show_bug.cgi?id=439312

I subscribed to the ticket.